packages/playwright/src/reporter.ts

```diff
@@ -403,6 +403,8 @@ export default class ReplayPlaywrightReporter implements Reporter {
           console.warn(`[replay.io]: ${line}`);
         });
       }
+
+      console.log("[replay.io]:");
```

Comment:
This looks really odd on its own. Maybe we could add some text to this, like this?
```suggestion
      console.log("[replay.io]: Completed");
```

Reply:
I don't think it looks odd since we already output some (almost) empty lines like this. I see this line as padding.

Reply:
This padding always bothered me - I intuitively expected to find some extra text below it. I think Brian had similar sentiment since he removed this recently [here](https://github.com/replayio/replay-cli/pull/586#discussion_r1662981854).

Perhaps it's the fact that this line is not completely empty (it has our prefix) makes is weird to me 🤷‍♂️

Reply:
I'd prefer if this were just be an empty line :) but it's at least better this way than for Playwright to trim it off.